extensions/wikia/Search/classes/Config.php

```diff
@@ -749,114 +857,139 @@ protected function bootstrapQueryService() {
 		if ( $this->getService()->getGlobal( 'EnableWikiaHomePageExt' ) ) {
 			$service = 'Select\\Dismax\\InterWiki';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Select\\Dismax\\InterWiki" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)